persistence/core/src/test/scala/com/lightbend/lagom/persistence/ActorSystemSpec.scala

```diff
@@ -54,6 +55,7 @@ abstract class ActorSystemSpec(system: ActorSystem)
   }
 
   val log: LoggingAdapter = Logging(system, this.getClass)
+  val coordinatedShutdown: CoordinatedShutdown = CoordinatedShutdown(system)
```

Comment:
Adding this field on `ActorSystemSpec` makes it easy to use on every spec on the PR. Probably we can followup on this change and remove any `coordinatedShutdown` instance that existed already in other subclasses of `ActorSystemSpec`.